src/firmware/stm32lib/deleteme_BU_ILD

```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Why is this file called deleteme? It seems to be asking to be deleted. Shall we grant it its wish?

Reply:
whoops, yep, deleted.